README-docker-compose.md

```diff
@@ -93,6 +93,22 @@
       wb:
         volumes_from:
           - container:wb-sync
+
+      wb_worker:
```

Comment:
no need to duplicate the config, these can be left in the override file commented out, further clarification can be added above if I did not explain it well enough.